src/std/text.ab

```diff
@@ -100,6 +100,17 @@ pub fun contains(text: Text, phrase: Text): Bool {
   return result == "1"
 }
 
+/// Checks if an array value is in the text.
+pub fun contains_multiple(text, terms): Bool {
```

Comment:
Function name doesn't clearly state what it does; I suggest renaming as `contains_any`, and perhaps providing a sister function `contains_all` while you're at it.

Reply:
We should also provide explicit types for functions, to create better compiler errors if someone passes (say) an integer argument:
```
pub fun contains_any(text: Text, terms: [Text]): Bool {
```